counter_test.go

```diff
@@ -3,8 +3,8 @@ package riaken_core
 import (
 	"testing"
 
-	"code.google.com/p/goprotobuf/proto"
-	"github.com/riaken/riaken-core/rpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/9gel/riaken-core/rpb"
```

Comment:
Thanks @bfosberry . I didn't know that the pull request is for a whole branch not up to a particular commit. Created a new branch and created a new pull request.